examples/thermostat/thermostat-common/src/thermostat-delegate-impl.cpp

```diff
@@ -244,3 +208,23 @@ void ThermostatDelegate::ClearPendingPresetList()
 {
     mNextFreeIndexInPendingPresetsList = 0;
 }
+
+std::optional<System::Clock::Milliseconds16> ThermostatDelegate::GetWriteTimeout(AttributeId attributeId)
+{
+    switch (attributeId)
+    {
+    case Presets::Id:
+        // If the client expects to edit the presets, then we'll give it 3 seconds to do so
```

Comment:
isn't 3 secs too less. it might work for automation but using chip-tool might need longer (assuming this is the timeout for actually editing the presets)

Reply:
same for schedules

Reply:
This was already bumped up per Boris' request. :)  I'll let David tell me how actual physical testing goes in terms of a valid maximum.

Reply:
still think it's not enough based on my testing with commands but will wait for what David finds during testing